.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,30 @@
+# Doc: https://github.com/redhat-plumbers-in-action/differential-shellcheck#usage
+---
+
+name: Differential ShellCheck
+on:
+  pull_request:
+    branches: [master]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    permissions:
+      security-events: write
+      pull-requests: write
```

Comment:
Is this needed?

Reply:
No, it's not. It is used for comments pointing to defects.

![image](https://user-images.githubusercontent.com/2879818/183250924-b24fdcf1-c10c-4e7a-b4e5-76f25c1e06a0.png)

It can be removed, and defects will still be marked in the Files changed tab. Also, comments aren't created when there are many defects (to keep the conversation "clean").

Reply:
The comments seem useful, let's keep `pull-requests: write`

Reply:
Although, if the feedback is sufficiently visible without `pull-requests: write` then that seems worth removing so we only have `contents: read` which is safe.

Reply:
Removed, @eregon PTAL